tox.ini

```diff
@@ -22,12 +22,12 @@ commands =
 [testenv:py27-functional]
 commands =
    python -V
-   {toxinidir}/scripts/kube-init.sh nosetests -v []
```

Comment:
Why only these two e2e tests? what about py36-functional and non-e2e tests? can we use py.test for those too?